.github/workflows/check-build-test.yml

```diff
@@ -107,8 +107,10 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - { java-version: 'temurin:1.8',  sbt-opts: '' }
-          - { java-version: 'temurin:1.11', sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
+          # other JDKs are covered in `scheduled-builds.yml`
+          - { jdkVersion: "1.8.0",  jvmName: "temurin:1.8.0",  extraOpts: '' }
+          - { jdkVersion: "1.11.0", jvmName: "temurin:1.11.0", extraOpts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
+          # { jdkVersion: "1.17.0", jvmName: "temurin:1.17.0", extraOpts: '' }
```

Comment:
I think this covers the JDK versions well enough but what about the Scala versions?

Reply:
Right I guess we need a `+` somewhere to run tests for all three Scala versions in the build.

Reply:
In several other projects we use `+Test/compile` in PR validation and only run tests with one Scala and JDK. I don't think it's worth running too many combinations in PR validation.

Reply:
That `+Test/compile` should be with JDK 8 (lowest supported), and that is not the case in this build.

Reply:
For the record "Check Code Compilation" currently runs `sbt +Test/compile; +It/compile` on JDK 11.